app/views/layouts/_main.html.slim

```diff
@@ -13,4 +13,5 @@
         div class="alert alert--#{severity}" = flash_message_for(flash[severity])
 
   main#main-content.govuk-main-wrapper role="main"
+    = render 'shared/browser_warning' if request.env['HTTP_USER_AGENT'] =~ /MSIE 8/
```

Comment:
I looked through the list of common user agents,
and it seems this is good enough.
(unless they run IE8 in IE6 or IE7 compatibility mode??)

https://developers.whatismybrowser.com/useragents/explore/software_type_specific/web-browser/6